library/sysroot/Cargo.toml

```diff
@@ -12,6 +12,10 @@ profiler_builtins = { path = "../profiler_builtins", optional = true }
 std = { path = "../std", public = true }
 test = { path = "../test", public = true }
 
+# Ferrocene addition: in bootstrap we need to set a libcore feature for code coverage testing, so we
+# need to add it as an explicit dependency here to do `--features core/$foo`.
+core = { path = "../core" }
```

Comment:
Why is it not optional?

Reply:
Is it because it is implicitly pulled in via `std` anyways?

Reply:
Yes, `core` should always be built when building the sysroot.